tests/dummy_qrc/qm_octave/platform.py

```diff
@@ -1,9 +1,9 @@
 import pathlib
 
-from qibolab.channel import Channel, ChannelMap
-from qibolab.instruments.dummy import DummyLocalOscillator as LocalOscillator
-from qibolab.instruments.qm import Octave, OPXplus, QMController
-from qibolab.platform import Platform
+from qibolab._core.channel import Channel, ChannelMap
+from qibolab._core.instruments.dummy import DummyLocalOscillator as LocalOscillator
+from qibolab._core.instruments.qm import Octave, OPXplus, QMController
+from qibolab._core.platform import Platform
```

Comment:
These platforms under `dummy_qrc` could also serve as template on how to write platforms, so I would prefer to stick to public objects. However, these particular instances are quite outdated and given that we postponed the test refactoring to a later release, instead of doing the effort to update them, I would suggest to just remove these files.

I believe that the tests using them are skipped anyway.